src/api/class-template-tags-request.php

```diff
@@ -56,25 +56,16 @@ private function get_template_tags() {
 			throw Api_Response_Error::message( $tags_response->get_error_message() );
 		}
 
-		$template_tags_json = wp_remote_retrieve_body( $tags_response );
+		$tags_decoded = json_decode( wp_remote_retrieve_body( $tags_response ), true );
 
-		$template_tags = [];
-
-		foreach ( json_decode( $template_tags_json, true ) as $tag_category => $tags ) {
-			switch ( $tag_category ) {
-				case 'Subject':
-					$template_tags['subject_tags'] = $tags;
-					break;
-
-				case 'Layout':
-					$template_tags['allowed_tags'] = $tags;
-					break;
-
-				default:
-					$template_tags['allowed_tags'] = array_merge( $template_tags['allowed_tags'], $tags );
-					break;
-			}
-		}
+		$template_tags = [
```

Comment:
Wow love how you replaced `foreach`. Usually I use `array_map` but I don't think I attempted to do that here because I had an array that had a key/value array type. This taught me something 🙂 

Can you just do a `return [ your code here ]` instead of assigning it to a variable and then returning the variable?

Reply:
I normally do that, but some people like keeping it assigned to a var for docblock parsing - wasn't sure if there was a preference, but I'm on the same page :slightly_smiling_face: